client/src/modules/plan/planner/idea/idea.js

```diff
@@ -0,0 +1,38 @@
+import angular from 'angular';
+
+// import child dependencies
+
+// imports for this component
+import template from './idea.html';
+import './idea.css';
+
+class IdeaController {
+  constructor(EventService, $stateParams) {
+    this.event = {
+      planId: 'hg5687h5834657h6',
+      title: 'John\'s best BBQ',
+      startTime: '2016-05-18T16:00:00Z',
+      endTime: '2016-05-18T16:00:00Z',
+      description: 'We will have a ton of fun at this park...',
+      imageUrl: 'https://d36tnp772eyphs.cloudfront.net/blogs/1/2014/08/Smith-Rock-940x595.jpg'
+    };
+    this.ideaId = $stateParams.ideaId;
+    this.idea = EventService.getEvent(this.ideaId);
+    this.EventService = EventService;
+  }
```

Comment:
add  //TODO: refactor EventService click-handlers to provide error handling if necessary